src/Tests/StringTests.cs

```diff
@@ -28,10 +28,12 @@ public void CamelCase()
         /// Test if pascal case works.
         /// </summary>
         [TestMethod]
-        public void PascalCase()
+        [DataRow("hello-world-test", "helloWorldTest")]
+        [DataRow("HelloWorldTest", "helloWorldTest")]
+        public void CamelCase(string input, string expected)
```

Comment:
Seems like you've removed the PascalCase test method, which should not have happened. If anything, you can add the CamelCase method while leaving the PascalCase method untouched. Also, I recommend adding a data row for a single letter to ensure it doesn't throw an error (since I see there is new logic that concatenates the first letter followed by everything else).

Reply:
Sure! It is done right now, please take a look